how-to/editor-in-chief-guide.md

```diff
@@ -1,164 +1,297 @@
+Copy the template below and use it in the review issue.
 
 ```{include} ../appendices/editor-in-chief-checks.md
 ```
 
-### 3. ✔️ Ensure that the package onboarding survey is filled out.
-
-Thank the authors graciously for filling out our survey. They can
-skip sections of it if they wish. We do need their contact
-information to stay in touch about package maintenance. We also
-want to track their experience with our review process and
-organization.
-
-### 4. ✔️ Assign an editor to the issue to manage the rest of the review and tag the issue with the `1/editor-assigned` tag in GitHub
-
-Once the package initial checks are complete, and it is determined that
-the package is in scope for pyOpenSci review, the Editor in Chief will assign an
-editor to the review issue and add the `1/editor-assigned` tag in GitHub.
-This may involve finding a new (guest) editor
-as described in the [onboarding guide](onboarding-guide.md).
-If you as Editor in Chief do recruit a new editor,
-be sure to complete all the onboarding steps described
-[here](onboarding-a-new-editor) so that the new editor
-has everything they need to manage the review,
-such as GitHub permissions and access to the relevant channels
-in the pyOpenSci Slack team.
-The editor will now begin the process of finding reviewers
-for the package.
-[Check out the editor guide for more information on the process that an editor follows](editors-guide.md)
-
-```{admonition} Diversity in the editorial & reviewer  team is important
+### 4. ✔️ Ensure that the package onboarding survey is complete
+
+Use [this spreadsheet](https://docs.google.com/spreadsheets/d/1jEk-DDpkz14Z07OX_o1cN2vHzVbJO6mQ83ihGXsWkLc/edit?gid=930774086#gid=930774086)
+to check whether the author—and ideally all maintainers—have completed the
+pre-review survey. You can search using their GitHub handle.
+
+Thank the author for completing the survey. While some sections are optional,
+we do need contact information to follow up about package maintenance. The
+survey also helps us track and improve our peer review process.
+
+### 5. ✔️  Assign an editor and add the `1/editor-assigned` label to the GitHub issue
+
+Once the package is determined to be in scope, the pre-review surveys are complete, and pre-review checks
+are done, it's time to assign an editor.
+
+Use the [editorial dashboard](https://www.pyopensci.org/metrics/peer-review/editorial-dashboard.html)
+to check which editors are currently available.
+
+Once you’ve selected an editor:
+
+1. Assign the issue to them in GitHub
+2. Update the YAML at the top of the issue with their GitHub username
+3. Add the `1/editor-assigned` label to the issue
+
+:::{note}
+If no current editor with relevant expertise is available, work with the peer
+review lead to recruit a guest editor or onboard someone new.
+
+Follow the [onboarding guide](onboarding-guide.md), and complete the full
+[onboarding process](onboarding-a-new-editor) to ensure they have:
```

Comment:
Should this have a `.md` on the end?

Though, I don't think this document exists in directory? https://github.com/pyOpenSci/software-peer-review/tree/cd31d5c80d0da0c319d6ed864b2b48a8faa240c5/how-to

Maybe this should be a URL segment added to the prior guide?

Reply:
oh it's a target. In Sphinx you can create targets like this:

(onboarding-a-new-editor)=

Above a header, it makes linking to sections in the guide easy, eliminating the need for long URLs with hashes. Good catch here tho!! it should work. I'll do a docs-test build to make sure before we merge.

Reply:
The target lives in the onboarding editors page. Sphinx targets are super nice :)